confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/PartitionInfo.java

```diff
@@ -484,6 +529,54 @@ public List<Pair<Long, TConsensusGroupId>> getSortedRegionSlotsCounter(
     return storageGroupPartitionTables.get(storageGroup).getSortedRegionSlotsCounter(type);
   }
 
+  /**
+   * Get total region number and update metric
+   *
+   * @param type SchemaRegion or DataRegion
+   * @return the number of SchemaRegion or DataRegion
+   */
+  private int getTotalRegionCountAndUpdateMetric(TConsensusGroupType type) {
+    Set<RegionGroup> regionGroups = new HashSet<>();
+    for (Map.Entry<String, StorageGroupPartitionTable> entry :
+        storageGroupPartitionTables.entrySet()) {
+      regionGroups.addAll(entry.getValue().getRegion(type));
+    }
+    int result = regionGroups.size();
+    // statistic
+    Map<TDataNodeLocation, Integer> dataNodeLocationIntegerMap = new HashMap<>();
```

Comment:
Add java doc to display what this Map stands for

Reply:
get